src/main/scala/cromwell/util/google/GoogleCloudStorage.scala

```diff
@@ -36,7 +36,13 @@ case class GoogleCloudStorage(client: Storage) {
   // See comment in uploadObject re small files. Here, define small as 2MB or lower:
   private val smallFileSizeLimit: Long = 2000000
 
-  def uploadObject(gcsPath: GoogleCloudStoragePath, inputStream: InputStream, byteCount: Long) = {
+  def uploadObject(gcsPath: GoogleCloudStoragePath, fileContent: String): Unit = {
+    val fileBytes = fileContent.getBytes
+    val bais = new ByteArrayInputStream(fileBytes)
```

Comment:
Sorry - this is my fault for phrasing it poorly. I actually meant the call to `uploadObject`, thinking that perhaps people would want to specify their own input stream. That said currently that's not the case so perhaps leaving it as is and waiting for something else to show up as a use case would be better.